### ISSUE TITLE
Feature: support named imports with aliases

### ISSUE THREAD

**Issue by SimonGolms** [azat-io/eslint-plugin-perfectionist#75]
### What rule do you want to change?

[`perfectionist/sort-named-imports`](https://eslint-plugin-perfectionist.azat.io/rules/sort-named-imports)

### Describe the problem

I would like to be able to influence the sorting of a single export from a module aka [named imports with aliases](https://www.typescriptlang.org/docs/handbook/modules.html#import).
Maybe this could be realized via another option, such as `ignore-alias`.

### Code example

Input:

```ts
import {
  Box,
  Dialog as MuiDialog,
  type DialogProps as MuiDialogProps,
  DialogActions,
  DialogContent,
  DialogTitle,
  Button,
  IconButton,
  Stack,
  Typography,
  type SxProps,
} from "@mui/material";
```

Output:
```ts
import {
  Box,
  Button,
  DialogActions,
  DialogContent,
  DialogTitle,
  IconButton,
  Dialog as MuiDialog,
  type DialogProps as MuiDialogProps,
  Stack,
  type SxProps,
  Typography,
} from "@mui/material";
```

Expected Output:
```js
// .eslintrc
{
  "plugins": ["perfectionist"],
  "rules": {
    "perfectionist/sort-named-imports": [
      "error",
      {
        "ignore-alias": true,
      }
    ]
  }
}
```
```ts
import {
  Box,
  Button,
  Dialog as MuiDialog,
  DialogActions,
  DialogContent,
  type DialogProps as MuiDialogProps,
  DialogTitle,
  IconButton,
  Stack,
  type SxProps,
  Typography,
} from "@mui/material";
```

### Validations

- [X] Read the [docs](https://eslint-plugin-perfectionist.azat.io).
- [X] Check that there isn't [already an issue](https://github.com/azat-io/eslint-plugin-perfectionist/issues) that reports the same bug to avoid creating a duplicate.

**Comment by azat-io** [comment]
Added in 4843f5b63d8a19ffe708d46d591c2af62aff522c.
I'll create a release tomorrow.

**Comment by azat-io** [comment]
Released in [v2.2.0](https://github.com/azat-io/eslint-plugin-perfectionist/releases/tag/v2.2.0).
